submissions/LGeorge007/DOM-practice/script.js

```diff
@@ -0,0 +1,175 @@
+let lastChoice;
+let startMenuPosition = 0;
+const content = document.querySelector("article");
+const navPanel = document.querySelector("nav");
+const menu = document.createElement("ul");
+
+
+const listOfUnits = [
+    {
```

Comment:
You can use a more readable data structure.
```suggestion
{
    feature: {
        title: 'Title',
        text: 'text'
     },
    tactics: {
        title: 'title',
        text: 'text'
     }
}
```